libmamba/include/mamba/api/configuration.hpp

```diff
@@ -569,6 +569,24 @@ namespace mamba
             node[name] = m_value;
         }
 
+        template <>
+        inline void ConfigurableImpl<fs::path>::dump_json(nlohmann::json& node,
```

Comment:
Not sure if it's the right way to do it. I suspect `nlohmann` to have a version that supports `std::filesystem::path` values, if there is, switching to it would make this change unnecessary, I believe.